src/3_PB/VerbaliInterni/VerbaleInterno_240315/content.typ

```diff
@@ -0,0 +1,55 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+Nell'incontro si è discussa l'organizzazione del lavoro futuro, di seguito se ne riportano i momenti salienti. 
+
+== Nuovi ruoli
+
+Vengono stabiliti i seguenti i nuovi incarichi relativi allo sprint corrente:
+- Caregnato Simone: Amministratore,Progettista, Programmatore,Verificatore;
+- Costantin Riccardo Alberto: Responsabile,Programmatore; 
+- D'Ovidio Giacomo: Programmatore; 
+- Kalaj Nancy: Programmatore, Verificatore, Progettista; 
+- Rango Matteo: Verificatore, Amministratore; 
+- Toniolo Riccardo: Verificatore.
+
+== Test di integrazione 
+
+Durante la riunione odierna, si è sollevata la questione dell'organizzazione dei test di integrazione e dell'effettiva utilità di questo processo all'interno del nostro progetto. Tuttavia, non si è giunta a una conclusione definitiva in merito, e la discussione è stata rimandata per consentire un'ulteriore valutazione. È stato evidenziato il bisogno di approfondire il ruolo dei test di integrazione nel contesto specifico del nostro progetto, valutando attentamente i potenziali benefici e svantaggi di questa pratica.
+
+== Modifiche all'_Analisi dei Requisiti_
+
+È stata discussa una modifica all'_Analisi dei requisiti_, riguardante la terminologia utilizzata per riferirsi alla sezione relativa alla visualizzazione della tabella relativa ai sensori. Si è deciso di modificare il termine "Dashboard Sensori" in "Dati grezzi". Se ne occupano i futuri Analisti.
+
+== Norme di progetto: _Manuale Utente_
+
+Si è deciso di aggiungere alle _Norme di Progetto_ la sezione riguardante il _Manuale Utente_. Questa sezione ha il duplice scopo di fungere da guida e di fornire una struttura organizzativa per lo sviluppo del documento del _Manuale Utente_. 
```

Comment:
```suggestion
Si è deciso di aggiungere alle _Norme di Progetto_ la sezione riguardante il _Manuale Utente_. Questa sezione ha il duplice scopo di fungere da guida e di fornire una struttura organizzativa per lo sviluppo del documento. Nancy Kalaj si occupa sia di normarne la struttura e i contenuti all'interno delle _Norme di Progetto_, che di procedere con la stesura iniziale del documento vero e proprio, riflettendo le funzionalità attualmente implementate all'interno della dashboard di visualizzazione Grafana.
```

Reply:
aggiungere un paio di paragrafi dedicati alle due dashboard che verrano aggiunte questo sprint, il contenuto (in breve) e da chi; più un altro paragrafetto dedicato a quel che farà Matteo nel PdQ (insomma, se sono state aperte nuove issue dovrebbero essere tracciabili qui)